services/auth/interface.go

```diff
@@ -18,8 +18,10 @@ type DataStore middleware.DataStore
 // SessionStore represents a session store
 type SessionStore session.Store
 
-// SingleSignOn represents a SSO authentication method (plugin) for HTTP requests.
-type SingleSignOn interface {
+// Auth represents an authentication method (plugin) for HTTP requests.
+type Auth interface {
```

Comment:
Lint would probably prefer this be named Method instead.

Reply:
But in fact, it doesn't complain.